pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: hypertrack_plugin
 description: Flutter plugin for HyperTrack, a cross-platform location tracking solution for Android and iOS
 repository: https://github.com/hypertrack/sdk-flutter
-version: 0.4.3
+version: 1.0.0
 homepage: https://www.hypertrack.com/
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.10.0"
+  flutter: ">=1.12.0"
 
 dependencies:
   flutter:
```

Comment:
I believe we could use 
`plugin_platform_interface: ^2.1.3`

Reply:
Similar motivation as [here](https://github.com/hypertrack/sdk-flutter/pull/27#discussion_r1006722078).

Reply:
I removed this dependency